jsapp/js/components/RESTServices/RESTServiceLogs.tsx

```diff
@@ -359,6 +382,3 @@ export default class RESTServiceLogs extends React.Component {
     }
   }
 }
-
-reactMixin(RESTServiceLogs.prototype, Reflux.ListenerMixin);
-reactMixin(RESTServiceLogs.prototype, mixins.contextRouter);
```

Comment:
what happened with mixins?

Reply:
`Reflux.ListenerMixin` is a weird mixin for listener callbacks. It makes the code "better"(?) by allowing to write:

```ts
// this:
this.listenTo(<listenable thing>, <callback>)
// instead of this:
<listenable thing>.listen(<callback>)
```

So when I remove it, all `listenTo` are changed to the normal code.

Reply:
As for `mixins.contextRouter` it was being used for getting the `assetUid` from the current path. Now I use `getRouteAssetUid` instead, so mixin can be removed.